vasyl_varyshchuk/Week_2/task_1/func_in_threads.py

```diff
@@ -0,0 +1,84 @@
+""""
+This module returns random size of file name from list by processing elements in separate threads
+"""
+
+import random
+import string
+from threading import Thread
+
+
+number_of_threads = 3
+length_of_list = number_of_threads * 100
```

Comment:
`length_of_list` is a bad name of variable.

Reply:
As in previous case, I modified name of variable and added this change to my last commit.